addons/killtracker/XEH_postInit.sqf

```diff
@@ -0,0 +1,129 @@
+    TRACE_1("no mission debriefing config",_this);
+};
+
+ACE_LOGINFO("Running Kill Tracking");
+
+// Variables:
+GVAR(eventsArray) = [];
+GVAR(outputText) = "None";
+
+// Add Event Handlers:
+[QGVAR(kill), {
+    params ["_name", "_killInfo"];
+    TRACE_2("kill eh",_name,_killInfo);
+    GVAR(eventsArray) pushBack format ["Killed: %1 %2", _name, _killInfo];
+    GVAR(outputText) = (GVAR(eventsArray) joinString "<br/>");
+}] call CBA_fnc_addEventHandler;
+
+[QGVAR(death), {
+    params ["_name", "_killInfo"];
+    TRACE_2("death eh",_name,_killInfo);
+    GVAR(eventsArray) pushBack format ["Died: %1 %2", _name, _killInfo];
+    GVAR(outputText) = (GVAR(eventsArray) joinString "<br/>");
+}] call CBA_fnc_addEventHandler;
+
+// Add Killed Event Handler - killed EH and lastDamageSource var are local only
+["CAManBase", "killed", {
+    params ["_unit", ["_killer", objNull]];
+    TRACE_2("killed",_unit,_killer);
+
+    private _killInfo = [];
+    if ((isNull _killer) || {_killer == _unit}) then {
+        private _aceSource = _unit getVariable ["ace_medical_lastDamageSource", objNull];
+        TRACE_1("",_aceSource);
+        if ((!isNull _aceSource) && {_aceSource != _unit}) then {
+            _killInfo pushBack "Last damage";
+            _killer = _aceSource;
+        };
+    };
+
+    // If killer is a vehicle get the commander (this is how vanilla does it?) and log the vehicle type
+    if ((!isNull _killer) && {!(_killer isKindof "CAManBase")}) then {
+        _killInfo pushBack format ["Vehicle: %1", getText (configfile >> "CfgVehicles" >> (typeOf _killer) >> "displayName")];
+        _killer = effectiveCommander _killer;
+    };
+
+    private _unitIsPlayer = hasInterface && {_unit == ace_player}; // isPlayer check will fail at this point
+    private _killerIsPlayer = (!isNull _killer) && {_unit != _killer} && {[_killer] call ACEFUNC(common,isPlayer)};
+    TRACE_2("",_unitIsPlayer,_killerIsPlayer);
+
+    // Don't do anything if neither are players
+    if (!(_unitIsPlayer || _killerIsPlayer)) exitWith {};
+
+    // Log firendly fire
+    if ((!isNull _killer) && {_unit != _killer}) then {
+        // side group is better, but group could already be null if it was last unit
+        private _unitSide = if (!isNull (group _unit)) then {side group _unit} else {side _unit};
+        private _killerSide = if (!isNull (group _killer)) then {side group _killer} else {side _killer};
+        if ([_unitSide, _killerSide] call BIS_fnc_areFriendly) then {
+            _killInfo pushBack "<t color='#ff0000'>Friendly Fire</t>";
+        };
+    };
+
+    // Log bleed out - ToDo: could change setDead to log the specific medical cause (e.g. blood loss / cardiac arrest / overdose)
+    private _bloodVolume = _unit getVariable ["ace_medical_bloodVolume", 100];
+    if (_bloodVolume <= 60) then {
+        _killInfo pushBack format ["Blood %1%2", floor _bloodVolume, "%"];
```

Comment:
I think I have done it before, but I forgot how to include % inside `format`s.
Tried `\%` and `%%`